library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -151,28 +151,28 @@ private void createDownloadsByBatchView(SQLiteDatabase db) {
      * columns to request from DownloadProvider.
      */
     public static final String[] DOWNLOAD_BY_BATCH_VIEW_COLUMNS = new String[]{
-            Downloads.Impl.DOWNLOADS_TABLE_NAME + "." + Downloads.Impl._ID + " AS _id ",
-            Downloads.Impl._DATA,
-            Downloads.Impl.COLUMN_MEDIAPROVIDER_URI,
-            Downloads.Impl.COLUMN_DESTINATION,
-            Downloads.Impl.COLUMN_URI,
-            Downloads.Impl.COLUMN_STATUS,
-            Downloads.Impl.DOWNLOADS_TABLE_NAME + "." + Downloads.Impl.COLUMN_DELETED,
-            Downloads.Impl.COLUMN_FILE_NAME_HINT,
-            Downloads.Impl.COLUMN_MIME_TYPE,
-            Downloads.Impl.COLUMN_TOTAL_BYTES,
-            Downloads.Impl.COLUMN_LAST_MODIFICATION,
-            Downloads.Impl.COLUMN_CURRENT_BYTES,
-            Downloads.Impl.COLUMN_NOTIFICATION_EXTRAS,
-            Downloads.Impl.COLUMN_BATCH_ID,
-            Downloads.Impl.Batches.COLUMN_TITLE,
-            Downloads.Impl.Batches.COLUMN_DESCRIPTION,
-            Downloads.Impl.Batches.COLUMN_BIG_PICTURE,
-            Downloads.Impl.Batches.COLUMN_VISIBILITY,
-            Downloads.Impl.Batches.COLUMN_STATUS,
-            Downloads.Impl.Batches.BATCHES_TABLE_NAME + "." + Downloads.Impl.Batches.COLUMN_DELETED,
-            Downloads.Impl.Batches.COLUMN_TOTAL_BYTES,
-            Downloads.Impl.Batches.COLUMN_CURRENT_BYTES
+            DownloadsTables.DOWNLOADS_TABLE_NAME + "." + DownloadsColumns._ID + " AS _id ",
+            DownloadsColumns.COLUMN_DATA,
+            DownloadsColumns.COLUMN_MEDIAPROVIDER_URI,
+            DownloadsColumns.COLUMN_DESTINATION,
+            DownloadsColumns.COLUMN_URI,
+            DownloadsColumns.COLUMN_STATUS,
+            DownloadsTables.DOWNLOADS_TABLE_NAME + "." + DownloadsColumns.COLUMN_DELETED,
+            DownloadsColumns.COLUMN_FILE_NAME_HINT,
+            DownloadsColumns.COLUMN_MIME_TYPE,
+            DownloadsColumns.COLUMN_TOTAL_BYTES,
+            DownloadsColumns.COLUMN_LAST_MODIFICATION,
+            DownloadsColumns.COLUMN_CURRENT_BYTES,
+            DownloadsColumns.COLUMN_NOTIFICATION_EXTRAS,
+            DownloadsColumns.COLUMN_BATCH_ID,
+            DownloadsColumnsBatches.COLUMN_TITLE,
```

Comment:
This is a view created through a join between the two tables so it's fair having both column definitions in one projection.